docs/migrations/ibc-migration-043.md

```diff
@@ -27,6 +27,66 @@ Feel free to use your own method for modifying import names.
 NOTE: Updating to the `v0.43.0` SDK release and then running `go mod tidy` will cause a downgrade to `v0.42.0` in order to support the old IBC import paths.
 Update the import paths before running `go mod tidy`.  
 
+## Chain Upgrades
+
+Chains may choose to upgrade via an upgrade proposal or genesis upgrades. Both in-place store migrations and genesis migrations are supported. 
+
+**WARNING**: Please read at least the quick guide for [IBC client upgrades](../ibc/upgrades/README.md) before upgrading your chain. It is highly recommended you do not change the chain-ID during an upgrade, otherwise you must follow the IBC client upgrade instructions.
+
+Both in-place store migrations and genesis migrations will:
+- migrate the solo machine client state from v1 to v2 protobuf definitions
+- prune all solo machine consensus states
+- prune all expired tendermint consensus states
+
+Chains must set a new connection parameter during either in place store migrations or genesis migration. The new parameter, max expected block time, is used to enforce packet processing delays on the receiving end of an IBC packet flow. Checkout the [docs](https://github.com/cosmos/ibc-go/blob/release/v1.0.x/docs/ibc/proto-docs.md#params-2) for more information.
+
+### In-Place Store Migrations
+
+The new chain binary will need to run migrations in the upgrade handler. The fromVM (previous module version) for the IBC module should be 1. This will allow migrations to be run for IBC updating the version from 1 to 2.
+
+Ex:
+```go
+app.UpgradeKeeper.SetUpgradeHandler("my-upgrade-proposal",
+        func(ctx sdk.Context, _ upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
+            // set max expected block time parameter. Replace the default with your expected value
+            // https://github.com/cosmos/ibc-go/blob/release/v1.0.x/docs/ibc/proto-docs.md#params-2
+            app.IBCKeeper.ConnectionKeeper.SetParams(ctx, ibcconnectiontypes.DefaultParams())
+
+            fromVM := map[string]uint64{
+                ... // other modules
+                "ibc":          1,
+                ... 
+            }   
+            return app.mm.RunMigrations(ctx, app.configurator, fromVM)
+        })      
+
+```
+
+### Genesis Migrations
+
+To perform genesis migrations, the following code must be added to your existing migration code.
+
+```go
+// add imports as necessary
+import (
+    ibcv100 "github.com/cosmos/ibc-go/modules/core/legacy/v100"
+    ibchost "github.com/cosmos/ibc-go/modules/core/24-host"
+)
+
+...
+
+// add in migrate cmd function
+// expectedTimePerBlock is a new connection parameter
+// https://github.com/cosmos/ibc-go/blob/release/v1.0.x/docs/ibc/proto-docs.md#params-2
+newGenState, err = ibcv100.MigrateGenesis(newGenState, clientCtx, *genDoc, expectedTimePerBlock)
+if err != nil {
+    return err 
+}
```

Comment:
tested, got as far as I could. Final confirmation is blocked on upstream changes (SDK is moving protobuf definition for upgrade/gov from v1beta1 to v1 and the current pr has minor bugs)

I confirmed the migration updated the pr, as far as I can tell, everything works on the IBC end, although I didn't get a chance to test that IBC clients were successfully upgraded. At the very least the chain started fine